pandemic/model.py

```diff
@@ -153,44 +162,32 @@ def pandemic(
-                chi_it = 1
-            # Southern Hemisphere & Fall/Winter Months
-            elif (origin['centroid_lat'] < 0 and time_step[-2:] in
-                      ['04', '05', '06', '07', '08', '09', '10']):
-                chi_it = 1
+            # check if time steps are annual (YYYY) or monthly (YYYYMM)
+            # if monthly, parse dates to determine if species is in the correct life cycle
+            # to be transported (set value to 1), based on the geographic location of the origin
+            # country (i.e., Northern or Southern Hemisphere)
+            if len(time_step) > 4:
+                if (
+                    origin["centroid_lat"] >= 0
+                    and time_step[-2:] not in season_dict["NH_season"]
+                ):
+                    chi_it = 0
+                elif (
+                    origin["centroid_lat"] < 0
+                    and time_step[-2:] not in season_dict["SH_season"]
+                ):
+                    chi_it = 0
+                else:
+                    chi_it = 1
             else:
-                chi_it = 0
+                chi_it = 1
 
             h_jt = destination["Host Percent Area"]
 
             if origin["Presence"] and h_jt > 0:
                 zeta_it = int(origin["Presence"])
 
-                origin_climates = origin.loc[['Af', 'Am',	'Aw',	'BWh', 'BWk', 
-                                              'BSh', 'BSk', 'Csa',	'Csb', 
-                                              'Csc', 'Cwa', 'Cwb', 'Cwc', 
-                                              'Cfa',	'Cfb', 'Cfc',	'Dsa', 
-                                              'Dsb',	'Dsc', 'Dsd',	'Dwa', 
-                                              'Dwb',	'Dwc', 'Dwd',	'Dfa', 
-                                              'Dfb',	'Dfc', 'Dfd',	'ET', 'EF']]
-
-                destination_climates = destination.loc[['Af', 'Am',	'Aw',	
-                                                        'BWh', 'BWk', 'BSh',
-                                                        'BSk', 'Csa',	'Csb',
-                                                        'Csc', 'Cwa', 'Cwb',
-                                                        'Cwc', 'Cfa',	'Cfb',
-                                                        'Cfc',	'Dsa', 'Dsb',
-                                                        'Dsc', 'Dsd',	'Dwa', 
-                                                        'Dwb',	'Dwc', 'Dwd',
-                                                        'Dfa', 'Dfb',	'Dfc',
-                                                        'Dfd',	'ET', 'EF']]
-                
-                delta_kappa_ijt = climate_similarity(
-                    origin_climates, destination_climates)
+                delta_kappa_ijt = climate_similarities[j, i]
 
                 if "Ecological Disturbance" in origin:
                     epsilon_jt = origin["Ecological Disturbance"]
```

Comment:
Is this maybe supposed to be epsilon_jt = destination["Ecological Disturbance"]?

Reply:
Good catch. Updating to reflect ecological disturbance in destination.